settings/teams.json

```diff
@@ -1,16 +1,4 @@
 {
-    "TeamAlpha": {
```

Comment:
Is there a reason this team is removed?

Reply:
I accidentally deleted while regression testing. I'll add it back in now.